### ISSUE TITLE
About the training accuracy.

### ISSUE THREAD

**Issue by ranery** [huawei-noah/AdderNet#12]
Hi, Hanting Chen,

I tried the same training setting (Poly LR schedule with 0.9 power; 400 epochs; 256 batch size), and I can only get 89.16% accuracy when training resnet20 on CIFAR-10, while the reported accuracy is 91.84%.

Could you provide more training details and help to explain the gap? thanks.

**Comment by HantingChen** [comment]
Do you use the full gradient and adaptive learning rate?

**Comment by ranery** [comment]
Yes, full gradient as Equ. 5 shows, adaptive learning rate as Equ. 12-13.

**Comment by ranery** [comment]
![image](https://user-images.githubusercontent.com/18202368/78089543-5f057e80-738d-11ea-9b5a-f5e0d0321b6e.png)

**Comment by HantingChen** [comment]
> 
> 
> I found that you use the poly LR schedule on CIFAR dataset, should I change to cosin LR schedule, I think that cannot incur 2% accuracy drop, is that any other setting I miss here?
> ![image](https://user-images.githubusercontent.com/18202368/78089543-5f057e80-738d-11ea-9b5a-f5e0d0321b6e.png)

Yes, the learning rate cannot incur 2% accuracy drop. I think there may be some mistake on your code. Do you use the same architecture with our code?

**Comment by ranery** [comment]
Your open source architecture only support for inference. For training, I have tried both python version and cuda version for acceleration as you suggested. Both of them show the similar results on resnet20. So I do not think there is bug on code, otherwise I cannot get 89.% accuracy.

Could you show the optimizer, LR setting and data augmentation etc training setting? I think that might be the reason, thanks!

**Comment by HantingChen** [comment]
Data prepare and optimizer is same as https://github.com/kuangliu/pytorch-cifar/blob/master/mai

We use the cosine LR with 0.1 as initial learning rate and 0 as final learning rate.

**Comment by JamesHujy** [comment]
Hi, Haoran,
Could you please release your training codes? I find it very hard to reproduce the experimental results.

**Comment by ranery** [comment]
> Hi, Haoran,
> Could you please release your training codes? I find it very hard to reproduce the experimental results.

Hi, I can not reproduce addernet with the similar accuracy reported in paper, instead resnet34 on imagenet is only 64.8% in my side, but the reported resnet18 can achieve 67%.
